cloudbuild.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
+# Build the container that will do the go build.
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['build', '-t', 'builder', '-f', 'Dockerfile.build', '.']
+# Do the go build.
+  - name: 'builder'
+    args: ['make', 'cross']
+# Upload to GCS.
+  - name: 'gcr.io/cloud-builders/gsutil'
+    args: ['cp', '-r', 'out/*', 'gs://container-diff/builds/$COMMIT_SHA/']
```

Comment:
Might be worthwhile to have a tag-watcher cloudbuild too?

Reply:
Sorry, what's a tag-watcher?

Reply:
Lol sorry - "trigger" was the word I was looking for.  Not sure where I got "tag-watcher"

Reply:
Still not following... :)

Reply:
My plan was to submit this then setup a trigger in the UI to run this for every commit to master. Is that what you're asking for?

Reply:
I was just suggesting that you add another cloudbuild.yaml using the same dockerfile, but triggered only when new tags are pushed (instead of every commit), with the step being slightly different:

`args: ['cp', '-r', 'out/*', 'gs://container-diff/releases/$TAG_NAME/']`

Reply:
I think thats what @nkubala was planning to do eventually?

Reply:
yeah I think we had originally talked about adding that. that way we can have the actual releases automated for us as well, as soon as the Github release gets published we get the binaries uploaded to the correct directories. we might also consider adding a 

```args: ['cp', '-r', 'out/*', 'gs://container-diff/releases/latest/']```

to that cloudbuild (to release the newest tag as `latest`), though we might want to have that in a different job so we can do rollbacks easily.

Reply:
Ah, yeah. I was thinking we'd have a separate release job to tag "latest" here.